app/src/main/java/com/keylesspalace/tusky/util/ImageLoadingHelper.kt

```diff
@@ -0,0 +1,43 @@
+@file:JvmName("ImageLoadingHelper")
+
+package com.keylesspalace.tusky.util
+
+import android.widget.ImageView
+import androidx.annotation.Px
+import com.bumptech.glide.Glide
+import com.bumptech.glide.load.resource.bitmap.FitCenter
+import com.bumptech.glide.load.resource.bitmap.RoundedCorners
+import com.keylesspalace.tusky.R
+
+
+private val fitCenterTransformation = FitCenter()
+
+fun loadAvatar(url: String?, imageView: ImageView, @Px radius: Int, animate: Boolean) {
```

Comment:
I would propose two changes to make this helper more useful and to cut down boilerplate:
- Let it decide on itself if it should animate or not. I don't think there's a case where it shouldn't decide that? It has a context and can figure it out. I don't know if there's any performance difference from fetching the pref each time, maybe it could be an object instead?
- Let it fetch the size in pixels by itself. It can either accept resource or some enum.

That's largely an ergonomic/DRY proposal, it's really not nice to fetch all these sizes and prefs each time you want to display an avatar.

Reply:
Imho, pixel size and pref should be loaded only once when the adapter is created for perf reasons, but yes its not DRY at all. Is it ok for you if I leave it like that for now?

Reply:
Absolutely, it's just a proposal. For perf reasons I proposed an an object which can be created once per holds or so (I think both dimensions and preys are cached)